app/models/chargeback.rb

```diff
@@ -1,4 +1,6 @@
 class Chargeback < ActsAsArModel
+  singleton_class.send(:prepend, DescendantLoader::ArDescendantsWithLoader)
```

Comment:
@lpichler Can you try autoloading Chargeback in that test before the describe tries to autoload it?

Before:
```
describe ChargebackContainerProject do
  before do
```

After:
```
Chargeback
describe ChargebackContainerProject do
  before do
```

I hate this but we need Chargeback loaded first to load the subclasses.  If a subclass tries to load first, it gets that circular dependency error because the subclass needs the base class to load, but the base class is trying to load the subclasses.  It's a mess.  Maybe @Fryguy @bdunne have suggestions.